numba/compiler.py

```diff
@@ -460,21 +462,31 @@ def _backend(self, lowerfn, objectmode):
         if self.library is None:
             codegen = self.targetctx.jit_codegen()
             self.library = codegen.create_library(self.bc.func_qualname)
-        lowered = lowerfn()
+        if config.DISABLE_JIT and not self.flags.force_enable_jit:
```

Comment:
If the JIT is disabled in decorator.py, is it necessary to also disable it here?